sherlock/resources/data.json

```diff
@@ -63,6 +63,14 @@
     "username_claimed": "blue",
     "username_unclaimed": "noonewouldeverusethis7"
   },
+  "9GAG": {
+    "errorType": "status_code",
+    "rank": 389,
+    "url": "https://www.9gag.com/u/{}",
+    "urlMain": "https://9gag.com/",
```

Comment:
Could you add `www` to the `urlMain` as well? I think it is better keep everything consistent :)

Reply:
I am not familiar with Regex, so that burden is on you. 😆